source/form.html

```diff
@@ -0,0 +1,210 @@
+            fill="#D22856" />
+        </svg>
+      </a>
+    </div>
+    <div class="page-footer__social social">
+      <h2 class="visually-hidden">Давайте дружить!</h2>
+      <ul class="social__list">
+        <li class="social__item">
+          <a href="" class="social__link social__link--twitter">
+            <img class="social__img" src="./img/icon-twitter.svg" width="18" height="14" alt="Иконка твиттер">
+          </a>
+        </li>
+        <li class="social__item">
+          <a href="" class="social__link social__link--facebook">
+            <img class="social__img" src="./img/icon-facebook.svg" width="9" height="18" alt="Иконка фейсбук">
+          </a>
+        </li>
+        <li class="social__item">
+          <a href="" class="social__link social__link--youtube">
+            <img class="social__img" src="./img/icon-youtube.svg" width="15" height="13" alt="Иконка ютуб">
+          </a>
+        </li>
+      </ul>
+    </div>
+    <div class="page-footer__copyright copyright">
+      <p class="copyright__text">Разработано</p>
+      <a class="copyright__link" href="https://htmlacademy.ru/intensive/adaptive">
+        <svg width="27" height="34" viewBox="0 0 27 34" fill="none" xmlns="http://www.w3.org/2000/svg">
+          <path
+            d="M13.6426 0.0157537L13.5 0L0 1.40733V26.0129L13.5 34L27 26.0129V1.40733L13.6426 0.0157537ZM25.2344 25.0151L13.5 31.9573L1.76561 25.0151V14.9293L13.4507 21.8451L13.4701 23.0984L5.45351 18.3548L5.44295 19.5801L13.4894 24.3885L13.5088 25.675L5.46408 20.9174L5.45351 22.1427L13.5 26.9511L21.6239 22.1182V20.8929V16.909L25.2432 14.7595L25.2344 25.0151ZM25.2344 13.4519L22.0147 15.3319L20.5325 16.2071L13.4666 12.0201L13.456 13.2454L19.4957 16.818L19.4429 16.8495L19.3126 16.9247L18.4536 17.4166L13.4666 14.4689L13.456 15.6942L17.3921 18.0222L16.4591 18.6314L16.438 18.6436L13.4666 16.902L13.456 18.1273L15.4029 19.2615L13.4155 20.4483L1.7973 13.5762L13.4349 6.61831L25.2344 13.4519ZM25.2344 12.2266L13.4401 5.31075L1.76561 12.2196V2.9897L13.5 1.76442L25.2344 2.9897V12.2266Z"
+            fill="#D22856" />
+        </svg>
+      </a>
+    </div>
+  </footer>
+  <div class="modal modal--error">
+    <div class="modal-wrapper">
+      <div class="modal__container">
+        <b class="modal__title">Что-то пошло не так!</b>
+        <p class="modal__text">Проверьте поля, выделенные красным, скорее всего вы забыли их заполнить</p>
+        <button class="button modal__button" type="submit">Ок</button>
```

Comment:
```suggestion
        <button class="button modal__button" type="button">Ок</button>
```
Формы здесь нет, а кнопка с типом `submit` отвечает за отправку данных из формы.